mixer/template/listentry/template.proto

```diff
@@ -48,5 +51,6 @@ option (istio.mixer.adapter.model.v1beta1.template_variety) = TEMPLATE_VARIETY_C
 // then the expression's [inferred type](https://istio.io/docs/reference//config/policy-and-telemetry/expression-language/#type-checking) must match the datatype of the field.
 message Template {
     // Specifies the entry to verify in the list.
```

Comment:
Please expand the comment to indicate this value can either be a string or an IP address.

Reply:
Thanks..Fixed